pkg/web/apps/apps_service_test.go

```diff
@@ -404,7 +404,7 @@ func Test_appsService_UpdateAppVersions(t *testing.T) {
 	}
 }
 
-func Test_appsService_CreateUpdateApp(t *testing.T) {
+func Test_appsService_CreateApp(t *testing.T) {
```

Comment:
Hi @craicoverflow you change the test to check just to create a new app.  However, we need to test the update as well. How be sure that this change will not affect it?